history.txt

```diff
@@ -1,6 +1,39 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
```

Comment:
The problem was actually bigger than the "...channel buttons turning black". It was "....channel buttons have no color and turning black" (we just fixed both problems in one go, but the one which we highlighted was the lesser one). I would not reformulate